utils/sampler/weighted_benchmark_test.go

```diff
@@ -159,7 +153,7 @@ func WeightedPowBenchmarkInitializer(
 
 func WeightedSingletonBenchmarkInitializer(b *testing.B, s Weighted, size int) {
 	weights := make([]uint64, size)
-	weights[0] = uint64(math.MaxInt64 - size + 1)
+	weights[0] = math.MaxUint64 - uint64(size+1)
```

Comment:
Why'd this go from `- size + 1` to `- size - 1` ?

Reply:
:eagle: :eye: